### PR TITLE
Add psucontrol_tendabeli

### PR DESCRIPTION
**Pull request by jgroman**

### CODE REVIEW

File: _plugins/psucontrol_tendabeli.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+- psucontrol
+- tenda
+compatibility:
+  os:
+  - posix
+  - windows
+  python: ">=2.7,<4"
+---
+
+See <https://github.com/kantlivelong/OctoPrint-PSUControl-TendaBeli> for information on configuration.
```
Comment (kantlivelong) [inline _plugins/psucontrol_tendabeli.md:25]: I didn't realize I wrote this. 😆

**Comment by kantlivelong** [comment]
Hi @ jgroman,

Everything seems good minus the one comment above. 

A link in https://github.com/jgroman/OctoPrint-PSUControl-TendaBeli/blob/master/README.md to the original source repo for your unofficial fork would be appreciated as well.

File: _plugins/psucontrol_tendabeli.md (hunk 1 above)
Comment (jgroman) [reply _plugins/psucontrol_tendabeli.md]: My apologies! Clearly my Ctrl-F skills are lacking :) 
Fixed.

Comment (kantlivelong) [reply _plugins/psucontrol_tendabeli.md]: All good 👍

**Comment by jgroman** [comment]
@kantlivelong README updated as well. Thanks for the heads up!

**Comment by kantlivelong** [comment]
> @kantlivelong README updated as well. Thanks for the heads up!

Hey thanks, appreciate it.

**Pull request merged**